test/codegen/golden/expected/ecommerce/app/routers/orders.py

```
@@ -0,0 +1,106 @@
+async def place_order(
+    order_id: str,
+    session: AsyncSession = Depends(get_session),
+) -> None:
+    svc = OrderService(session)
+    return await svc.place_order(order_id)
+
+@router.post("/orders/{order_id}/payments", status_code=201)
+async def record_payment(
+    order_id: str,
+    body: OrderCreate,
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)